src/AccordionContainer/AccordionContainer.js

```diff
@@ -67,16 +67,15 @@ class AccordionContainer extends Container<StoreState> {
                 items,
             };
         });
-    }
+    };
 
-    removeItem(key: string | number) {
-        return this.setState(state => ({
+    removeItem = (key: string | number) =>
+        this.setState(state => ({
             items: state.items.filter(item => item.uuid !== key),
         }));
-    }
 
-    setExpanded(key: string | number, expanded: boolean) {
-        return this.setState(state => ({
+    setExpanded = (key: string | number, expanded: boolean) =>
```

Comment:
Ran into a javascript context issue where `this` didn't equal what I wanted it to equal, so using arrow functions here to bind the correct context.